### PR TITLE
feat(opamp): Add support for new source templates (elastic, mysql, postgres, rabbitmq, redis)

### PR DESCRIPTION
**Pull request by rnishtala-sumo**
Add support for new source templates 
- elastic
- mysql
- postgres
- rabbitmq
- redis

### CODE REVIEW

File: pkg/extension/opampextension/testdata/opamp.d/opamp-rabbitmq-config.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+receivers:
+  rabbitmq:
+    endpoint: http://localhost:15672
```
Comment (chan-tim-sumo) [inline pkg/extension/opampextension/testdata/opamp.d/opamp-rabbitmq-config.yaml:3]: is ```http://``` required for rabbitmq?

Comment (chan-tim-sumo) [reply pkg/extension/opampextension/testdata/opamp.d/opamp-rabbitmq-config.yaml:3]: whoops, nvm found it in the readme. that's so weird 😆

Comment (rnishtala-sumo) [reply pkg/extension/opampextension/testdata/opamp.d/opamp-rabbitmq-config.yaml:3]: No its optional - https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/rabbitmqreceiver/README.md

Comment (rnishtala-sumo) [reply pkg/extension/opampextension/testdata/opamp.d/opamp-rabbitmq-config.yaml:3]: the endpoint field is optional. But the scheme needs to be specified

**Review by echlebek — APPROVED** [review]

**Pull request merged**